pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
```

Comment:
I mean, 
Original McCabe rating is 7
```bash
pylint -d all -e too-complex --max-complexity=1 checkers/modules_odoo.py | grep _get_duplicate_xml_fields
checkers/modules_odoo.py:473: [R1260(too-complex), ModuleChecker._get_duplicate_xml_fields] '_get_duplicate_xml_fields' is too complex. The McCabe rating is 7
```

Applying the following patch the mccabe Rating is 9:
```diff
diff --git a/pylint_odoo/checkers/modules_odoo.py b/pylint_odoo/checkers/modules_odoo.py
index 0c8ff89..fd08ed5 100644
--- a/pylint_odoo/checkers/modules_odoo.py
+++ b/pylint_odoo/checkers/modules_odoo.py
@@ -486,9 +486,10 @@ class ModuleChecker(misc.WrapperModuleChecker):
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
             parents = [e.getparent() for e in items]
-            same_parent = [
-                item for item, count in collections.Counter(parents).items()
-                if count > 1]
+            same_parent = []
+            for item, count in collections.Counter(parents).items():
+                if count > 1:
+                    same_parent.append(item)
             # Remove elements which are in different parent element
             # Parents in items but not in same_parent list
             exclude = list(set(parents) - set(same_parent))
```
```bash
pylint -d all -e too-complex --max-complexity=1 checkers/modules_odoo.py | grep _get_duplicate_xml_fields
checkers/modules_odoo.py:473: [R1260(too-complex), ModuleChecker._get_duplicate_xml_fields] '_get_duplicate_xml_fields' is too complex. The McCabe rating is 9
```